tests/integration/mocked_services.py

```diff
@@ -144,3 +144,4 @@
 url_notify_party_and_respondent_account_locked = f'{app.config["PARTY_URL"]}/party-api/v1/respondents/edit-account-' \
                                                  f'status/{party["id"]}'
 url_banner_api = f"{app.config['BANNER_SERVICE_URL']}/banner"
+url_respondent_party_by_id = f'{app.config["PARTY_URL"]}/party-api/v1/respondents/id/{party["id"]}'
```

Comment:
This didn't need to be added.  In the same file there is `url_get_respondent_party` which is identical.  Use this instead in your tests and remove this new one that you've added.

Reply:
Actually I don't need this. you are right